Doc/4547 keyword conventions (#4559)
ansys/pyaedt: doc/source/Getting_started/Contributing.rst

  Validate GitHub Workflows................................................Passed
  blacken-docs.............................................................Passed

Naming conventions
~~~~~~~~~~~~~~~~~~
Consistency of names helps improve readability and
ease of use. Starting with release 0.8 a concerted effort
has been made to
improve consistency of naming and adherence to
:ref:`PEP-8<https://peps.python.org/pep-0008/>`_.

For example, methods used to create or access entities in

> Electronics Desktop require that a name to be passed to the method or function

Reviewer: ```suggestion
AEDT require that a name be passed to the method or function
```

as an argument.
It is tempting to

> include context as part of that variable name. For example, one is tempted to use
> ``setupname``
> as an argument to :meth:`Hfss.create_setup`_.
> However, the context "setup" is
> explicitly defined by the method name. The variable ``name`` provides
> a more compact
> description of the variable in this context.

Reviewer: ```suggestion
include context as part of that variable name. For example, while it is tempting to use
``setupname``
as an argument to :meth:`Hfss.create_setup`_,
the context "setup" is
explicitly defined by the method name. The variable ``name`` provides
a more compact
description of the variable in this context.
```

> One may
> also find both ``setup_name`` and ``setupname`` used in previous versions of PyAEDT
> for various methods or classes.

Reviewer: ```suggestion
In previous PyAEDT versions, you can also find both ``setup_name`` and ``setupname`` used 
for various methods or classes.
```

Improving naming consistency improves maintainability and readability.

The following table illustrates the recommended conventions:

> .. list-table:: Keywords and Object Names

Reviewer: ```suggestion
.. list-table:: Keywords and object names
```
Sentence case for all headings/tites.

   :widths: 25 25 50
   :header-rows: 1

>    * - Old Name
>      - New Name

Reviewer: ```suggestion
   * - Old name
     - New name
```

     - Example
   * - ``setupname``, ``setup_name``, ``sweepname``
     - ``name``
     - ``Hfss.create_setup()``, ``Hfss.create_linear_step_sweep()``
   * - ``usethickness``
     - ``thickness``
     - ``Hfss.assign_coating()``
   * - ``entities``
     - ``assignment``
     - ``Maxwell.assign_current_density()``
   * - ``entity_list``
     - ``assignment``
     - ``Maxwell.assign_symmetry()``

> Please take care to use descriptive names for
> variables and classes that adhere to PEP-8 and are consistent with conventions already
> used in PyAEDT.

Reviewer: ```suggestion
Take care to use descriptive names for
variables and classes that adhere to PEP-8 and are consistent with conventions already
used in PyAEDT.
```
Avoid "please" in technical doc.
